Adyen/Core/Models/CardType.swift

```diff
@@ -141,7 +141,7 @@ public enum CardType: RawRepresentable, Codable, Equatable, Hashable {
     /// Fallback option for any other scheme name
     case other(named: String)
     
-    // swiftlint:disable cyclomatic_complexity
+    // swiftlint:disable cyclomatic_complexity function_body_length
```

Comment:
This `function_body_length ` seems to be not needed
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=1468&id=Adyen_adyen-ios&open=AYv8kz--mvDt7x1qhA6U

Reply:
Apparently sonar has a different configuration than our local linter which has a limit on 40 lines 🤔 
will leave it for now and do another PR with changes that address sonarcloud smells 👌